test/test.googleauth.ts

```diff
@@ -1021,30 +1017,30 @@ it('_checkIsGCE should not set the _isGCE flag when not running on GCE',
      scope.done();
    });
 
-it('_checkIsGCE should retry the check for isGCE if it fails the first time',
+it('_checkIsGCE should retry the check for isGCE on transient http errors',
```

Comment:
So honest question - do we think this test belongs here?  I could see the argument for deleting the test, and trusting the tests in `gcp-metadata`.

Reply:
Same question for the tests below.

Reply:
Some of these tests do belong here – specifically the tests that ensure we cache the answer from the first time around. Others can be potentially removed, but I don't have strong feelings. I can remove them if you prefer.

Reply:
🤷‍♂️ s'all good